src/app/registration/registration.component.css

```diff
@@ -1,39 +1,42 @@
+::ng-deep :root {
+  --link-to-log-in-page-text-color: #459fed;
+  --btn-to-sign-up-color: #3899ec;
+}
+
 a {
-  color: #459fed;
+  color: var(--link-to-log-in-page-text-color);
   text-decoration: none;
 }
-.title1 {
+
+.container--for-page-description {
   display: block;
-  text-align: center !important;
-  font-size: 35px;
-}
-.title1 > * {
-  top: 150px;
+  text-align: center;
+  font-size: 2.2rem;
 }
-.title2 {
-  font-size: 20px;
+
+.header--for-registration-page {
+  padding-top: 6.25rem;
 }
-.example-container {
-  display: block;
-  text-align: center;
+
+.paragraph--to-question-about-acc {
```

Comment:
avoid to and for, because these words do not give additional info, but make naming longer